### ISSUE TITLE
Add vertical alignment

### ISSUE THREAD

**Issue by dominz88** [diegodario88/quake-terminal#38]
This is helpful when you want to access a subjacent application that is hidden by the terminal by clicking such application. I''m aware that this es less quake-like and more like a toggle option, but it would be nice to have it.

**Comment by diegodario88** [comment]
could you elaborate on that?

**Comment by dominz88** [comment]
Hello,

The terminal is always attached to the top of the screen, i'd like to have the terminal floating, without touching any of the borders.

![image](https://github.com/user-attachments/assets/c4175a3c-826b-4b20-974b-25d0809eae40)

As shown in the image, it would allow users to click on the underlying window, in this case the web browser.  

If implemented, the vertical alignment would be: bottom, center (as in the image) and bottom.

**Comment by diegodario88** [comment]
Hey, just a heads-up. You should be able to click around if "auto hide" is turned off.

**Comment by tflori** [comment]
@diegodario88 please consider this feature again. for example: I hide the title bar of konsole with rounded window borders and the window begins 48px lower...and the top bar actually has a "transparent border".. I would like the terminal to start there... so I would need a vertical adjustment of about -54px

![Image](https://github.com/user-attachments/assets/66033ccb-abad-4b13-b4bf-6ed13b7b46e0)

**Comment by raleik-pl** [comment]
Yes pleeease add vertical alignment! This is the best extension ever! I'm using Tilix which is awesome terminal, but doesn't support own "quake-mode" in wayland. Thankfully, your extension makes it work flawlesly! ❤️ 
rel https://github.com/diegodario88/quake-terminal/issues/84
